tools/fds/server.cpp

```diff
@@ -26,10 +26,11 @@
 #include <condition_variable>
 #include <csignal>
 
-#include <fastrtps/Domain.h>
-#include <fastrtps/participant/Participant.h>
+#include <fastdds/dds/domain/DomainParticipant.hpp>
+#include <fastdds/dds/domain/DomainParticipantFactory.hpp>
 #include <fastdds/dds/log/Log.hpp>
 
+using namespace eprosima::fastdds::dds;
 using namespace eprosima;
 using namespace fastrtps;
 using namespace std;
```

Comment:
Do we need these?

Reply:
Yes, DomainParticipantQos in dds namespace